### PR TITLE
pd: better Debug format

### PR DESCRIPTION
**Pull request by overvenus**
Address the comment https://github.com/pingcap/tikv/pull/1712#discussion_r108858534

PTAL @andelf @BusyJay @siddontang

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @disksing

File: src/pd/async/client.rs [hunk 1]
```diff
@@ -59,17 +59,17 @@ impl RpcClient {
...
     }
 }
 
 impl fmt::Debug for RpcClient {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        write!(fmt,
-               "PD gRPC Client connects to cluster {:?}",
-               self.cluster_id)
+        fmt.debug_struct("RpcClient")
+            .field("cluster_id", &self.cluster_id)
```
Comment (andelf) [inline src/pd/async/client.rs]: self.cluster_id

Comment (siddontang) [inline src/pd/async/client.rs]: any update? @overvenus

Comment (overvenus) [inline src/pd/async/client.rs]: It expects reference.
```
error[E0308]: mismatched types
  --> src/pd/async/client.rs:70:34
   |
70 |             .field("cluster_id", self.cluster_id)
   |                                  ^^^^^^^^^^^^^^^ expected reference, found u64
   |
   = note: expected type `&std::fmt::Debug`
              found type `u64`
   = help: try with `&self.cluster_id`

error[E0308]: mismatched types
  --> src/pd/async/client.rs:71:30
   |
71 |             .field("leader", self.get_leader())
   |                              ^^^^^^^^^^^^^^^^^ expected reference, found struct `kvproto::pdpb::Member`
   |
   = note: expected type `&std::fmt::Debug`
              found type `kvproto::pdpb::Member`
   = help: try with `&self.get_leader()`
```

**Comment by disksing** [comment]
LGTM

**Pull request merged**